sdk/insights/ci.yml

```diff
@@ -0,0 +1,32 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger:
+  branches:
+    include:
+    - master
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/insights/
+
+pr:
+  branches:
+    include:
+    - master
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/insights/
+
+stages:
+- template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: insights
+    ArtifactName: packages
+    Artifacts:
+    - name: Azure.Management.Template
```

Comment:
This should be Azure.ResourceManager.Insights that match the folder and namespace.